jobs/config.json

```diff
@@ -3233,6 +3233,30 @@
       "sig-testing"
     ]
   },
+  "ci-kubernetes-e2e-gce-scale-coredns": {
+    "args": [
+      "--cluster=gce-scale-cluster",
+      "--env=CLUSTER_DNS_CORE_DNS=true",
+      "--env=HEAPSTER_MACHINE_TYPE=n1-standard-8",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-common.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-scalability-highspeed-common.env",
+      "--env-file=jobs/env/ci-kubernetes-e2e-gce-large-performance.env",
+      "--extract=ci/latest",
+      "--gcp-master-image=gci",
+      "--gcp-node-image=gci",
+      "--gcp-nodes=3",
+      "--gcp-project=kubernetes-scale",
+      "--gcp-zone=us-east1-a",
+      "--provider=gce",
+      "--test_args=--ginkgo.focus=\\[Feature:PerformanceDNS\\] --gather-resource-usage=master --gather-metrics-at-teardown=master",
+      "--timeout=510m",
+      "--use-logexporter"
+    ],
+    "scenario": "kubernetes_e2e",
+    "sigOwners": [
+      "sig-scalability"
```

Comment:
I'd prefer it being owned by the relevant vertical, which in this case is sig-network.